suites/upgrade/firefly-hammer-x/stress-split/06-monb-upgrade-hammer/monb.yaml

```diff
@@ -0,0 +1,6 @@
+tasks:
+- ceph.restart:
+    daemons: [mon.b]
+    wait-for-healthy: false
+    wait-for-osds-up: true
+- print: "**** done restarted mon.b 06-monb-upgrade-hammer/monb.yaml" 
```

Comment:
need to upgrade the other 3 osds, and the other mon, and mds here, right?  everything to hammer?